Makefile

```diff
@@ -159,8 +135,13 @@ bin-prometheus: mkdir
 #<<<
 bin-traefik: mkdir
 	echo "    started bin-traefik" >> $(LOGFILE)
+ifeq ($(VENV_OS), arm64mac)
+	curl -Lo $(DIR)/bin/miniconda_install.sh \
+		https://github.com/traefik/traefik/releases/download/v$(TRAEFIK_VERSION)/traefik_v$(TRAEFIK_VERSION)_darwin_arm64.tar.gz
```

Comment:
and the filename contains `darwin_amd` not `darwin_arm`

Reply:
No, this one is supposed to be darwin_arm; that's the M1 mac.

Reply:
Doh. There is no arm version for 2.5.0, so we need to update to https://github.com/traefik/traefik/releases/download/v2.5.1/traefik_v2.5.1_darwin_arm64.tar.gz

Reply:
Right. The darwin_arm version doesn't appear until traefik 2.5.1, so need to add a note to docs that M1 users need to update TRAEFIK_VERSION in their .env to at least 2.5.1 (alternately, we could update the traefik version in the example .env file).